mmv1/templates/terraform/examples/iam_workforce_pool_provider_extra_attributes_oauth2_config_client_basic.tf.erb

```diff
@@ -12,8 +12,8 @@ resource "google_iam_workforce_pool_provider" "<%= ctx[:primary_resource_id] %>"
     "google.subject" = "assertion.sub"
   }
   oidc {
-    issuer_uri        = "https://accounts.thirdparty.com"
-    client_id         = "client-id"
+    issuer_uri        = "https://sts.windows.net/826602fe-2101-470c-9d71-ee1343668989/"
+    client_id         = "https://analysis.windows.net/powerbi/connector/GoogleBigQuery"
```

Comment:
could you explain how this follows the intended use case better?

Reply:
The extra_attributes_oauth2_config was recently added for Power BI groups support, see https://cloud.google.com/iam/docs/workforce-sign-in-power-bi#create-workforce-provider. For AZURE_AD_GROUPS_MAIL attribute type, it should have Power BI config or would not work when signing in.